Sources/Actions/Admin/ACP.php

```diff
@@ -1615,6 +1615,7 @@ public static function getFileVersions(array &$versionOptions): array
 			$directories += ['template_versions' => Theme::$current->settings['theme_dir']];
 		}
 
+		/** @var string $dirname */
```

Comment:
The IDE is thinking that $dirname` is an array in its static analysis.  This gives it a hint that it's a string.  Note that this will help psalm as well when we implant it to understand the variable.  We can also do `/** @var string[] $directories */` earlier in the code.  The IDE (and static analysis) will also understand then its an array of strings.